### PR TITLE
Update user guide, user info, developer guide

### PR DESCRIPTION
**Pull request by LolfoollorS**
Supercede previous PR

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -320,6 +320,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
 
+* **Medical Administrator**: A person who oversees, plan, direct, and coordinate health care delivery.
+* **Patients**: A person receiving or registered to receive medical treatment.
+* **Nurses**: A person trained to care for the sick or infirm, especially in a hospital.
+* **HealthCare Xpress**: A desktop app for managing patients that require home-visits.
```
Comment (xhphoong) [inline docs/DeveloperGuide.md]: Maybe the 'c' here should be consistent. should be small 'c' I guess? cuz the readme.md there is small 'c' for Healthcare Xpress

File: docs/UserGuide.md [hunk 2]
```diff
@@ -87,6 +87,19 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
+### 2. Adds a nurse to the Healthcare Xpress book
+
+Format:`add c/N n/NAME p/PHONE_NUMBER e/EMAIL g/GENDER [t/TAG]`
+
+> 💡 Tip: A nurse can have any number of tags (including 0).
+> You may type it in any order.
+> To add a nurse, type c/N specifically.
```
Comment (xhphoong) [inline docs/UserGuide.md]: Maybe for here can make it as point form and list it down instead of 3 tips is in 1 continuous line.

**Review by xhphoong — COMMENTED** [review]
Overall good just a bit minor changes maybe needed?

**Review by ou4jairmeiogusrjejiepr — COMMENTED** [review]
LGTM, can just implement xh's suggestions and merge, thanks

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -320,6 +320,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
 
+* **Medical Administrator**: A person who oversees, plan, direct, and coordinate health care delivery.
+* **Patients**: A person receiving or registered to receive medical treatment.
+* **Nurses**: A person trained to care for the sick or infirm, especially in a hospital.
```
Comment (johnbenedictyan) [inline docs/DeveloperGuide.md]: Do we think that patients and nurses are universally known terms? Maybe we can further contextualise the meaning of patients to those are require home visits?

**Review by johnbenedictyan — COMMENTED** [review]
LGTM other than the terms patients and nurses

**Pull request merged**